client/lbclient/boot/replication.js

```diff
@@ -23,17 +23,32 @@ module.exports = function(client) {
   };
 
   // setup model replication
+  var since = {push: -1, pull: -1};
```

Comment:
> Is there a style guide for the style you describe?

https://strongloop.atlassian.net/wiki/display/SL/Code+Style+Guide

We have had some discussions about spacing in array/object initialisation - see https://strongloop.atlassian.net/wiki/display/SL/Code+Style+Guide?focusedCommentId=21430339#comment-21430339